### PR TITLE
Galachain connect MVP

### PR DESCRIPTION
**Pull request by befitsandpiper**
Basic functionality around connecting to metamask and sending transactions on chain using new Connect Client class.

### CODE REVIEW

File: project.json [hunk 1]
```diff
@@ -8,6 +8,14 @@
       "options": {
         "lintFilePatterns": ["**/*.ts"]
       }
+    },
```
Comment (herr-bubu) [inline project.json:11]: not needed, we don't use verdaccio

File: package.json [hunk 2]
```diff
@@ -49,7 +52,8 @@
     "ts-node": "^10.9.2",
     "typedoc": "^0.25.4",
     "typedoc-plugin-markdown": "^4.0.0-next.22",
-    "typescript": "5.3.3"
+    "typescript": "5.3.3",
+    "verdaccio": "^5.0.4"
```
Comment (herr-bubu) [inline package.json:56]: we don't use verdaccio

File: connect/package.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@gala-chain/connect",
+  "version": "0.0.1",
```
Comment (herr-bubu) [inline connect/package.json:3]: I assume the version will be aligned to what we currently have (1.1.17)
@dzikowski wdyt?

File: connect/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# connect
```
Comment (herr-bubu) [inline connect/README.md:1]: Please add some info about the project itself

File: .verdaccio/config.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# path to a directory with all packages
```
Comment (herr-bubu) [inline .verdaccio/config.yml:1]: -> 🗑️

File: package.json [hunk 6]
```diff
@@ -49,7 +52,8 @@
...
   },
   "nx": {
     "includedScripts": []
```
Comment (herr-bubu) [inline package.json:59]: there's a workspace section, please add this project there too :)

File: connect/package.json (hunk 3 above)
Comment (dzikowski) [reply connect/package.json:3]: Yes, and we should add it to ensure versions script

File: package.json [hunk 7]
```diff
@@ -30,9 +30,12 @@
     "@nx/js": "17.3.2",
     "@nx/linter": "17.3.2",
     "@nx/workspace": "17.3.2",
+    "@swc-node/register": "~1.6.7",
+    "@swc/core": "~1.3.85",
+    "@swc/helpers": "~0.5.2",
```
Comment (dzikowski) [inline package.json:35]: what is the purpose of those libraries?

File: chain-connect/src/GalachainConnectClient.ts [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+  private async submit(signedPayload: Record<string, any>, chaincodeUrl: string, method: string) {
+    // TODO: use gateway rather than ops
+
+    // Note: GalaChain Uri maybe should be constructed based on channel and method,
+    // rather than passing full url as arg
+    // ie `${baseUri}/api/${channel}/token-contract/${method}`
+    const url = `${chaincodeUrl}/${method}`;
+    const response = await fetch(url, {
+      method: "POST",
+      body: JSON.stringify(signedPayload),
```
Comment (dzikowski) [inline chain-connect/src/GalachainConnectClient.ts:73]: we should use `serialize` from `@gala-chain/api` here

**Comment by herr-bubu** [comment]
looks good. Linting fails in `connect` with unused library but apart from that everything is good 🚀

**Pull request merged**